docs/modules/cli/pages/integration-revealjs.adoc

```diff
@@ -0,0 +1,56 @@
+= Integrate with reveal.js
+
+In this section, you'll learn:
+
+* [x] How to install the required dependencies.
+* [x]  What attributes you must define.
+* [x] How to convert an AsciiDoc file to a _reveal.js_ presentation.
+
+== Prerequisites
+
+We assume that you already have a fully working version of Asciidoctor.js CLI installed and available in your `PATH`.
+If this is not the case, please read the xref:setup/install.adoc[installation instructions].
+
+To convert an AsciiDoc file to a _reveal.js_ presentation, we will need to install the following module:
+
+[source]
+$ npm i asciidoctor-reveal.js
+
+NOTE: If you prefer Yarn over npm, you can install Asciidoctor.js reveal.js using `yarn add asciidoctor-reveal.js`
+
+Please remember that if you want this converter to be available anywhere on your system,
+you will have to install this module with the `--global` option with `npm` or the `global` command with Yarn.
+
+== Convert an AsciiDoc presentation into slides
+
+Once this module is installed, we create a file named _presentation.adoc_ with the following content:
+
+.presentation.adoc
+```adoc
```

Comment:
I think we should use a consistent authoring style for code blocks. Let's use the formal syntax:

```
[source,asciidoc]
----
----
```